app/views/notes/_comment.html.erb

```diff
@@ -201,18 +208,22 @@
 </script>
 
 <style>
-  .open>#emoji-dropdown.dropdown-menu {
+  .open {
```

Comment:
Could we actually make a more specific css classname here? `.open` seems likely to collide with another stylesheet somewhere, someday... you know? What do you think?

Reply:
Correct we should!

Reply:
I have changed the class name from `.open` to `.emoji-container`.
Please review.
Thank you!